packages/e2e-tests/specs/editor/plugins/block-variations.test.js

```diff
@@ -1,191 +0,0 @@
-/**
- * WordPress dependencies
- */
-import {
-	activatePlugin,
-	createNewPost,
-	deactivatePlugin,
-	insertBlock,
-	searchForBlock,
-	pressKeyWithModifier,
-	openDocumentSettingsSidebar,
-	togglePreferencesOption,
-	toggleMoreMenu,
-	canvas,
-} from '@wordpress/e2e-test-utils';
-
-describe( 'Block variations', () => {
-	beforeAll( async () => {
-		await activatePlugin( 'gutenberg-test-block-variations' );
-	} );
-
-	beforeEach( async () => {
-		await createNewPost();
-	} );
-
-	afterAll( async () => {
-		await deactivatePlugin( 'gutenberg-test-block-variations' );
-	} );
-
-	const expectInserterItem = async ( blockTitle ) => {
-		const inserterItem = await page.$x(
-			`//button[contains(@class, 'block-editor-block-types-list__item')]//span[text()="${ blockTitle }"]`
-		);
-		expect( inserterItem ).toBeDefined();
-		expect( inserterItem ).toHaveLength( 1 );
-	};
-
-	test( 'Search for the overridden default Quote block', async () => {
-		await searchForBlock( 'Quote' );
-
-		expect( await page.$( '.editor-block-list-item-quote' ) ).toBeNull();
-		expectInserterItem( 'Large Quote' );
-	} );
-
-	test( 'Insert the overridden default Quote block variation', async () => {
-		await insertBlock( 'Large Quote' );
-
-		expect(
-			await canvas().$(
-				'.wp-block[data-type="core/quote"] blockquote.is-style-large'
-			)
-		).toBeDefined();
-	} );
-
-	test( 'Insert the Large Quote block variation with slash command', async () => {
```

Comment:
I removed this test during migration. The previous test case uses the slash command, so we already cover this case.